css/style.css

```diff
@@ -203,8 +206,8 @@ aside h2 {
 /* ----------- UI - Top Bar ----------- */
 
 main {
-  display: grid;
-  grid-template-rows: 10% 60% 30%;
```

Comment:
I would say let's have `display: grid` here. Those toggles and filters will surely come soon after the internship.